tensorflow/BUILD

```diff
@@ -63,9 +63,16 @@ package_group(
     packages = ["//tensorflow/..."],
 )
 
-sh_binary(
+filegroup(
     name = "swig",
-    srcs = ["tools/swig/swig.sh"],
+    srcs = select({
+        "//tensorflow:windows" : [
+	    "tools/swig/swig.bat"
+        ],
+        "//conditions:default": [
+            "tools/swig/swig.sh"
+        ],
+    }),
```

Comment:
Just so I understand, do we need this condition? (I'd have thought that since we're running on MSYS we could just reuse `tools/swig/swig.sh` on Windows.)